index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+    />
+  </head>
+  <body>
+    <header id="header">
+      <div class="logo">
+        <a href="#"
+          ><h1><span>K</span>-Kal</h1></a
+        >
+      </div>
+      <nav class="nav">
+        <div class="nav-container">
+          <a href="#menu" class="nav-menu"
+            ><i class="fa-solid fa-bars nav-icon"></i
+          ></a>
+          <a href="#" class="nav-menu nav-menu-second"
+            ><i class="fa-solid fa-x nav-icon"></i
+          ></a>
+          <ul class="side-bar" id="menu">
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <main>
+      <section class="section-p1">
+        <div class="div1">
+          <h2>I'm Cristhian. Glad to see you!</h2>
+          <p>
+            I'm a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don't hesitate to
+            contact me.
+          </p>
+        </div>
+        <div class="icons">
+          <p>lest's connect</p>
+          <a href="https://twitter.com/Kal24031992"><img src="/img/Twitter.svg" alt="" /></a>
+          <a href="https://www.linkedin.com/in/cristhian-castillo-41136322a/"><img src="/img/Linkedin.svg" alt="" /></a>
+          <a href="#"><img src="/img/M.svg" alt="" /></a>
+          <a href="https://github.com/Kal2403"><img src="/img/github.svg" alt="" /></a>
+          <a href="#"><img src="/img/thougn.svg" alt="" /></a>
```

Comment:
- [ ] Nice wok implementing the icons :clap: But since it is a list of icons, it would be better for semantic reasons, to use `ul` tag to group them and `li` tag per icon and each has a link (`a` tag) to link to the wright platform.